rr_custom_msgs/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ add_service_files(
         rr_gazebo/SpawnBoxRandom.srv
         rr_node_tools/RemoveFile.srv
         rr_node_tools/String.srv
-        rr_node_tools/Int32.srv
+        rr_node_tools/Uint32.srv
```

Comment:
I would suggest we change the capitalization of this file to match standard ros messages (UInt32): http://docs.ros.org/en/api/std_msgs/html/msg/UInt32.html